WooCommerce/WooCommerceTests/ViewRelated/Dashboard/CardReaderSettings/CardReaderSettingsConnectedViewModelTests.swift

```diff
@@ -160,17 +160,35 @@ final class CardReaderSettingsConnectedViewModelTests: XCTestCase {
         XCTAssert(analytics.receivedEvents.contains(WooAnalyticsStat.cardReaderSoftwareUpdateTapped.rawValue))
     }
 
-    func test_startCardReaderUpdate_ViewModel_LogsTracksEvent_cardReaderSoftwareUpdateStarted() {
+    func test_card_reader_update_starts_viewModel_logs_tracks_event_cardReaderSoftwareUpdateStarted() {
         // Given
+        // .available not sent
 
         // When
         mockStoresManager.simulateUpdateStarted()
 
         // Then
         XCTAssert(analytics.receivedEvents.contains(WooAnalyticsStat.cardReaderSoftwareUpdateStarted.rawValue))
+        XCTAssert(analytics.receivedProperties.contains(where: {
+            $0["software_update_type"] as! String == "Required"
```

Comment:
nit: it might be nice not crashing the tests if this unwrapping fails 😄 (same for the other cases in the PR)
```suggestion
            $0["software_update_type"] as? String == "Required"
```